commitizen/commands/init.py

```diff
@@ -109,17 +120,37 @@ def _ask_tag_format(self, latest_tag) -> str:
                 tag_format = "$version"
         return tag_format
 
-    def _install_pre_commit_hook(self):
+    def _search_pre_commit(self):
+        return shutil.which("pre-commit") is not None
+
+    def _exec_install_pre_commit_hook(self, hook_types: List[str]):
+        if not hook_types:
+            raise ValueError("At least 1 hook type should be provided.")
+        cmd_str = "pre-commit install " + "".join(
+            f"--hook-type {ty}" for ty in hook_types
+        )
```

Comment:
I found that I forgot to add a space between `--hook-type` options...
@Lee-W Do you have any idea how to write tests for this?
Because it depends on `pre-commit`'s behavior, it's harder to test.
```suggestion
        cmd_str = "pre-commit install " + " ".join(
            f"--hook-type {ty}" for ty in hook_types
        )
```

Reply:
we could verify whether the code generate the pre-commit command we want. e.g., mock the caller function of `cmd_str` to check